experimental/adaptive-dialog/javascript_nodejs/01.multi-turn-prompt/package.json

```diff
@@ -16,11 +16,10 @@
         "url": "https://github.com/Microsoft/BotBuilder-Samples.git"
     },
     "dependencies": {
-        "botbuilder": "4.9.2",
-        "botbuilder-dialogs": "4.9.2",
-        "botbuilder-dialogs-adaptive": "4.9.2-preview",
-        "botbuilder-lg": "4.9.2",
-        "adaptive-expressions": "4.9.2",
+        "botbuilder": "4.11.0-rc1",
```

Comment:
let's we wait for the final release and update the package version to that.

Reply:
agreed.  we don't update package.json's across the samples repo until after SDK's have been released.